pyseext/field_helper.py

```diff
@@ -202,13 +204,17 @@ def get_field_value(self, form_cq: str, name: str) -> Any:
         Args:
             form_cq (str): The component query that identifies the form panel in which to look for the field
             name (str): The name of the field
+            is_date (bool): Indicates whether the field is a date time or not. Defaults to False.
```

Comment:
Did you remove this parameter?
Don't you need it to convert to a Python date?

Reply:
Yes sorry should have removed this, will do now. As above I don't think we need to convert to date as it's already in the correct format. Although saying that I wonder if there are other date fields not in that format, what will happen then. Will investigate further

Reply:
I suppose my point is that the field value is a date not a string, so this method is not actually matching the type that the value is.

Now you have fixed the format you should be able to convert easily.
That said, it sounds like the raw value might be easier for you to represent in checks.

I don't feel that strongly about it, so happy with whatever you decide.

Cheers

Reply:
Is what is returned from the JS side not a string as toLocaleString() which is used on the JS side if field is a date returns a string? Or have I misunderstood what I thought I was doing, which is highly likely :)

Reply:
Sorry, penny finally dropped for my pea brain. Have updated so it converts back to datetime obj if date field. And like you previously mentioned can use raw value when asserting a field value